test/Atmos/EDMF/edmf_kernels.jl

```diff
@@ -77,6 +78,9 @@ function vars_state(::Environment, ::Primitive, FT)
     @vars(atke::FT, aθ_liq_cv::FT, aq_tot_cv::FT, aθ_liq_q_tot_cv::FT,)
 end
 
+vars_state_filtered(target::SpecificEDMFFilter, FT) =
+    vars_state(target.atmos, Primitive(), FT)
```

Comment:
Is this supposed to be Primitive() or Prognostic() ?

Reply:
its a shortcut to get definitions of the density free variables as defined in Primitive

Reply:
`Primitive` is correct-- we want the `NamedTuple` type to exclude `ρ`

Reply:
We've discussed this since and, actually, we need to spell this out because `Primitive` has pressure (instead of energy?), and we want specific total internal energy.